### DISCUSSION TITLE
now dev - subfolder for react project

### DISCUSSION THREAD

**Discussion by reflash** [vercel/vercel#4127] [General]
Hi,

Is it possible to deploy using now dev having my react in a subfolder?
I know it should be possible to do that using custom builds section, but what is the original one that's used by default? I was trying to replicate same behavour, but it fails to run react and apis on the same port

**Comment by coetry** [comment]
Hi @reflash you should be able to specify your application root when you import your project. Do you have a public repo that we can look at?

**Reply by coetry** [reply]
I misunderstood your question. You're not talking about deploying, but rather invoking `now dev` when your react app is in a subfolder. For this scenario, I would recommend you invoke the native dev command directly. `now dev` is more suitable for when you have Serverless Functions and your framework doesn't support it.

**Reply by reflash** [reply]
I'm using serverless functions + react. I'm building my app using one of the examples (functions are in /api and frontend is in src). What I was trying to do is to put frontend in a folder with its own package.config, but in that case `now dev` doesn't pick it up for build, it just finds serverless functions.
